integration/tests/basic/rpc/test_rpc_estimate_gas.py

```diff
@@ -19,6 +21,11 @@ class TestRpcEstimateGas:
     accounts: EthAccounts
     web3_client: NeonChainWeb3Client
 
+    # geth for tag.EARLIEST returns 
```

Comment:
it happens because we don't have history on test stands
Let's remove this comment

Reply:
fixed ✅